### ISSUE TITLE
Lights not showing after calling GVRExternalScene to load new X3D scene

### ISSUE THREAD

**Issue by mwitchwilliams** [Samsung/GearVRf#1202]
Using X3D and the Anchor tag, I call GVRExternalScene to load a new X3D scene (file).  The old X3D scene is correctly removed.  And the new X3D is animating as expected, however, the lights are not showing.

**Comment by mwitchwilliams** [comment]
For testing, use the Pull Request
https://github.com/Samsung/GearVRf/pull/1215

with the demo below.  Begin with anchorVP02.x3d and click on the orange sphere.
A new scene (animation01.x3d) will appear with an animation.
The Directional and Point lights can be commented out in animaton01.x3d and the textured objects will appear because we can render a scene without lights.  But with the lights in the scene, there the objects appear black against a dark blue background.
[GVRExternalScene-lighting issue.zip](https://github.com/Samsung/GearVRf/files/1032599/GVRExternalScene-lighting.issue.zip)

**Comment by liaxim** [comment]
@mwitchwilliams Is this problem still present?

**Comment by mwitchwilliams** [comment]
Yes, this is still an issue.  Nola should be added to this issue.
